openapi.yml

```
@@ -0,0 +1,383 @@
+openapi: "3.0.1"
+info:
+  title: Notifications
+  version: "2"
+x-amazon-apigateway-gateway-responses:
+  BAD_REQUEST_BODY:
+    responseTemplates:
+      application/json: |
+        {
+          "error": "$context.error.validationErrorString"
+        }
+x-amazon-apigateway-request-validators:
+  full:
+    validateRequestBody: true
+    validateRequestParameters: true
+x-amazon-apigateway-request-validator: full
+security:
+  - TenantAuthorizer: []
+paths:
+  /notifications/org:
```

```suggestion
  /notifications/organization:
```

We've been spelling out organization everywhere